### ISSUE TITLE
twisted.python.usage.Options documentation has improperly rendering code blocks

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#7329]
|[<img alt="Julian's avatar" src="https://avatars.githubusercontent.com/u/329822?s=50" width="50" height="50">](https://github.com/Julian)| @Julian reported|
|-|-|
|Trac ID|trac#7329|
|Type|defect|
|Created|2014-05-11 01:23:55Z|
|Branch|https://github.com/twisted/twisted/tree/usage-docstring-formatting-7329|

The examples on this page render literally rather than preformatted.

E.g.:

optFlags is assigned a list of lists. Each list represents a flag parameter, as so:

| optFlags = [['verbose', 'v', 'Makes it tell you what it doing.'], | ['quiet', 'q', 'Be vewy vewy quiet.']]

<details><summary>Searchable metadata</summary>

```
trac-id__7329 7329
type__defect defect
reporter__Julian Julian
priority__normal normal
milestone__ 
branch__branches_usage_docstring_formatting_7329 branches/usage-docstring-formatting-7329
branch_author__rwall rwall
status__closed closed
resolution__fixed fixed
component__core core
keywords__documentation_easy documentation easy
time__1399771435000000 1399771435000000
changetime__1416157447389549 1416157447389549
version__None None
owner__exarkun exarkun

```
</details>

**Comment by twisted-trac** [comment]
|<img alt="fbrennen's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50"><a name="note_1"></a>|fbrennen commented|
|-|-|

Added a patch for this in github. Find the pull request here: [https://github.com/twisted/twisted/pull/48]

**Comment by twisted-trac** [comment]
|[<img alt="wallrj's avatar" src="https://avatars.githubusercontent.com/u/978965?s=50" width="50" height="50">](https://github.com/wallrj)<a name="note_2"></a>|@wallrj commented|
|-|-|

(In [43149]) Branching to usage-docstring-formatting-7329.

**Comment by twisted-trac** [comment]
|[<img alt="wallrj's avatar" src="https://avatars.githubusercontent.com/u/978965?s=50" width="50" height="50">](https://github.com/wallrj)<a name="note_3"></a>|@wallrj commented|
|-|-|

I merged fbrennen's PR into a twisted branch and pushed the build button.

Ready for review.

**Comment by twisted-trac** [comment]
|<img alt="adamtheturtle's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50"><a name="note_4"></a>|adamtheturtle commented|
|-|-|

Note that [#7683](https://github.com/twisted/twisted/issues/7683) overlaps this issue and also has a patch for (some of?) these page rendering issues.

**Comment by twisted-trac** [comment]
|[<img alt="adiroiban's avatar" src="https://avatars.githubusercontent.com/u/204609?s=50" width="50" height="50">](https://github.com/adiroiban)<a name="note_5"></a>|@adiroiban commented|
|-|-|

[[[#7683](https://github.com/twisted/twisted/issues/7683)](https://github.com/twisted/twisted/issues/7683)](https://github.com/twisted/twisted/issues/7683) is fine. do we also need this patch?

Maybe is better to apply these changes in #7683 and merge them together in #7683

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_6"></a>|@exarkun commented|
|-|-|

> [[#7683](https://github.com/twisted/twisted/issues/7683)](https://github.com/twisted/twisted/issues/7683) is fine. do we also need this patch?

This work was done first.  Perhaps you should ask if we also need #7683, given the work done here?

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_7"></a>|@exarkun set owner to @exarkun<br>@exarkun set status to `assigned`|
|-|-|

> I merged fbrennen's PR into a twisted branch and pushed the build button.

Thanks Richard!

And thanks for this fix fbrennen.  Sorry about the long delay on getting it reviewed.

These changes look good to me apart from the whitespace introduced before `::` and the new `:` introduced to make a `:::`.  Both of those seem to make rendering of the API documentation worse.

I'm going to back out those changes, make sure the rendering of the examples still looks good, and then merge.  Thanks again.

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_8"></a>|@exarkun set status to `closed`|
|-|-|

(In [43485]) Merge usage-docstring-formatting-7329

Fixes: [#7329](https://github.com/twisted/twisted/issues/7329)
Author: fbrennen
Reviewer: exarkun

Fix invalid epytext markup around the API documentation examples for
`twisted.python.usage.Options`.
